### ISSUE TITLE
BTT not seeing MS Sculpt Comfort mouse

### ISSUE THREAD

**Issue by bomtailey** [folivoraAI/BetterTouchTool#2628]
**IMPORTANT:** Don't post private requests here (e.g. license issues, etc.). Please send me an email at andreas@folivora.ai instead. 

---

#### Description of bug/feature request/question:
Please only post one bug or feature request per issue.

BTT suddenly stopped seeing the mouse.  I've used BTT for a couple of years and never had a problem with it defining mouse actions and suddenly, it just doesn't know the mouse is present.

#### Affected input device (e.g. MacBook Trackpad, Magic Mouse/Trackpad, Touch Bar, etc.):

Microsoft Sculpt Comfort mouse

#### Device information:

Type of Mac:MacBook Air - Mid 2012
macOS version: 10.13.4
BetterTouchTool version: 2.467

#### Additional information (e.g. StackTraces, related issues, screenshots, workarounds, etc.):

---

**Note:** Before bug reporting, please make sure you have the latest version of BetterTouchTool and that you have already tried to restart your system :-). If you encounter a crash, please attach a crash log from the macOS Console.app from the "User Diagnostic Reports" section.

**Comment by fifafu** [comment]
There hasn't been a change to this.
Is your license still valid for this version of BTT (if not you should have received a popup telling you)

**Comment by bomtailey** [comment]
Hi Andreas, just wanted to follow up, I did a whole sequence of things and now BTT is responding normally to the mouse (though mouse input doesn’t show up in Live View (I don’t remember if it ever did).  Maybe because it’s not a Magic Mouse?).  As is usually the case, I did so many things, I don’t know what was responsible for the change.  The only thing missing is that I’ve done something to Karabiner so at the moment, the Windows button on my MS mouse isn’t being read but, aside from that, I have all the BTT mouse functionality that I have defined so that’s major for me.
Thanks again for your responsiveness and your app.

Tom Bailey
